app.js

```diff
@@ -3,11 +3,49 @@
 //Welcome
 var userName = prompt ('Before we play the About Chuck guessing game, please enter your name.');
 alert('Welcome ' + userName + '! Lets get started.' );
+var questions = ['True or False, Chuck\'s full name is Charles Forsythe?',
+'True or False, Chuck once lived in Colorado?',
+'True or False, Chuck Chuck is a painter by training?',
+'True or False, Chuck a father?',
+'Chuck like collecting tools, especially wood working tools. Can you guess how many hand planes Chuck owns?',
+'Can you guess one of Chuck\'s favorite things to eat?',
+'True or False, Chuck wants to work for you?'];
```

Comment:
When you initialize an array across multiple lines, you'll want to keep indent the values on their own lines:

```javascript
var questions = [
  'Uno',
  'Dos',
  'Tres'
];
```